core/src/main/java/org/hibernate/ogm/type/impl/ByteStringType.java

```diff
@@ -28,7 +28,7 @@ public ByteStringType() {
 
 	@Override
 	public String getName() {
-		return "byte_integer";
+		return "byte_as_string";
 	}
```

Comment:
almost deserves a separate JIRA / PR ;)

I'm reviewing the 7th commit and no change so far seems related to OGM-366, you could have pulled all these out in smaller changes & quicker integrations.